Assets/Scripts/Projects/ProjectData.cs

```diff
@@ -241,5 +242,15 @@ public static void SetCurrentProject(ProjectData initialProjectTemplate)
             current = initialProjectTemplate;
             current.RootLayer = new RootLayer("RootLayer");
         }
+
+        /// <summary>
+        /// Recursively collect all assets from each of the property data elements of every layer for loading and
+        /// saving purposes. 
+        /// </summary>
+        /// <returns>A list of assets on disk</returns>
+        public IEnumerable<Asset> GetAssets()
+        {
+            return rootLayer.GetAssets();
```

Comment:
waarom moet je hier in ProjectData alle assets getten? moet niet elke LayerData op het moment dat die geladen wordt en een Asset heeft deze asset inladen (of in de queue zetten om ingeladen te worden)

Reply:
Deze functie is bedoeld om zometeen in een keer alle assets te verzamelen bij het opslaan van het project en functioneert als een soort AssetRegistry. Ik kan dit ook lostrekken in een aparte service als dat het duidelijker maakt